lib/index.js

```diff
@@ -65,10 +65,15 @@ Optimizely.prototype.initialize = function() {
 };
 
 /**
- * Track. The Optimizely Web trackEvent can only send one property, revenue
+ * Track. The Optimizely Classic Web trackEvent accepts a payload object.
  *
- * https://www.optimizely.com/docs/api#track-event
- * http://developers.optimizely.com/javascript/reference/#api-function-calls
+ * Optimizely Classic: https://developers.optimizely.com/classic/javascript/reference/index.html#api-function-calls
+ * Optimizely X:       https://developers.optimizely.com/x/solutions/javascript/reference/index.html#function_setevent
+ *
+ * This API is forward compatible from Optimizely Classic to Optimizely X.
+ *   - Classic will only utilize the revenue key of the payload
```

Comment:
this logic is in optimizely backend then? or is this happening in the JS sdks for classic/x? just curious :) pretty neat!